<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quanLy</class>
 <widget class="QDialog" name="quanLy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>466</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>182</y>
     <width>117</width>
     <height>208</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>500</height>
    </size>
   </property>
   <property name="title">
    <string>Command</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushNV">
      <property name="text">
       <string>Nhân Viên</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushMENU">
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushKV">
      <property name="text">
       <string>Khu Vực</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushBC">
      <property name="text">
       <string>Báo Cáo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushBack">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>800</width>
     <height>181</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>400</height>
    </size>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Index"/>
   <widget class="QWidget" name="pageNhanVien">
    <widget class="QTableView" name="_tableViewNV">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>631</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupNV">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="_nvradcc">
        <property name="text">
         <string>Chấm Công</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_nvradxoa">
        <property name="text">
         <string>Xóa</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_nvradsua">
        <property name="text">
         <string>Sửa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_nvradthem">
        <property name="text">
         <string>Thêm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pageMenu">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="_mradxoa">
        <property name="text">
         <string>Xóa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="_mradsua">
        <property name="text">
         <string>Sửa</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="_mradthem">
        <property name="text">
         <string>Thêm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="_tableViewM">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>651</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageKhuVuc">
    <widget class="QTableView" name="_tableViewKV">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>471</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>119</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="_kvradthem">
        <property name="text">
         <string>Thêm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_kvradsua">
        <property name="text">
         <string>Sửa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_kvradxoa">
        <property name="text">
         <string>Xóa</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>631</width>
     <height>201</height>
    </rect>
   </property>
   <widget class="QWidget" name="index_2"/>
   <widget class="QWidget" name="pageThem">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>621</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Insert</string>
     </property>
     <widget class="QLineEdit" name="_lineiname">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineiid">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_linei3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Thêm</string>
      </property>
     </widget>
     <widget class="QLabel" name="_ilaid">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="_ila3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="_ilaname">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="_ila4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="_ila5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>80</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_linei4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_linei5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pageSua">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>621</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Update</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Sửa</string>
      </property>
     </widget>
     <widget class="QLabel" name="_ulaid">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MaNV: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineuid">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineuname">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineu3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineu4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="_ula4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>HeSoLuong:</string>
      </property>
     </widget>
     <widget class="QLabel" name="_ulaname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TenNV: </string>
      </property>
     </widget>
     <widget class="QLabel" name="_ula3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NamSinh: </string>
      </property>
     </widget>
     <widget class="QLabel" name="_ula5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineu5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pageChamCong">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Salary</string>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Chấm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MaNV:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SoGioLam:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineSmanv">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="_lineSgl">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pageXoa">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>621</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Delete</string>
     </property>
     <widget class="QLabel" name="_dlaid">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MaNV: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="_linedxoa">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>631</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Index_BC"/>
   <widget class="QWidget" name="pageBC">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>611</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>492</x>
        <y>340</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Xuất File Excel</string>
      </property>
     </widget>
     <widget class="QTableView" name="_tableViewBC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>581</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Xóa Báo Cáo</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
